### ISSUE TITLE
module compiled with Swift 5.0.1 cannot be imported by the Swift 5.1 compiler

### ISSUE THREAD

**Issue by Insfgg99x** [jitsi/jitsi-meet#4367]
I get an error in Xcode 11.0 beta 2 (11M337n):

module compiled with Swift 5.0.1 cannot be imported by the Swift 5.1 compiler: /Users/xgf/Desktop/cw/Third/JitsiMeet/JitsiMeet.framework/Modules/JitsiMeet.swiftmodule/x86_64.swiftmodule

would plz. help me fix this issue

**Comment by saghul** [comment]
The SDK is compiled with a stabkle version of Xcode, version 10.2 to be exact. You must compile if yourself if you are using a beta version of Xcode.

**Comment by TannerJuby1** [comment]
@saghul and what if you are getting this error with the latest version of Xcode (11.0)??

**Comment by saghul** [comment]
There is an open issue about this already. Now that Xcode 11 is out of beta, we'll fix it. Patches are weldcome, of course :-)

**Comment by pushpsenairekar2911** [comment]
Hello @saghul, Can we get an exact timeline where can we new 'Jitsimeet.framework' with Xcode 11 Support. We have tried by compiling repo using Xcode but we are getting

 ```
/bin/sh -c /Users/pushpsenairekar/Library/Developer/Xcode/DerivedData/jitsi-meet-cmjgqokaxkdqemdtllqijmjxaqxo/Build/Intermediates.noindex/ArchiveIntermediates/JitsiMeet/IntermediateBuildFilesPath/sdk.build/Release-iphoneos/JitsiMeet.build/Script-DE3D81D6228B50FB00A6C149.sh
/Users/pushpsenairekar/Library/Developer/Xcode/DerivedData/jitsi-meet-cmjgqokaxkdqemdtllqijmjxaqxo/Build/Intermediates.noindex/ArchiveIntermediates/JitsiMeet/IntermediateBuildFilesPath/sdk.build/Release-iphoneos/JitsiMeet.build/Script-DE3D81D6228B50FB00A6C149.sh: line 14: pushd: /Users/pushpsenairekar/Library/Developer/Xcode/DerivedData/jitsi-meet-cmjgqokaxkdqemdtllqijmjxaqxo/Build/Intermediates.noindex/ArchiveIntermediates/JitsiMeet/IntermediateBuildFilesPath/sdk.build/Release-iphoneos/JitsiMeet.build/../ios: No such file or directory

** ARCHIVE FAILED **
```

**Comment by saghul** [comment]
I can't provide an exact timeline. Out of curiosity: why do you need Xcode 11?

**Comment by pushpsenairekar2911** [comment]
We are creating a new project with SwiftUI support which includes calling. We are using Jitsi in the same.

**Comment by saghul** [comment]
I see. Sorry, but you seem to have no other choice than to wait for us to fix it... or come up with the fix and send it to us :-)

**Comment by pushpsenairekar2911** [comment]
@saghul  Thanks for your support. We are able to build the framework successfully.

**Comment by olcayertas** [comment]
@pushpsenairekar2911 can you tell us how did you compile SDK?

**Comment by saghul** [comment]
@olcayertas See https://github.com/jitsi/jitsi-meet/pull/4707/files/d76ea00c5f824d810290c10b9f50901a02c58b4f..3e1fb67555fae811ce5bf8501c16cdd0bf820300

**Comment by saroar** [comment]
is it fixed it? :(

**Comment by saghul** [comment]
THis is fixed already in master and SDK version 2.4.0
